pkg/sql/information_schema.go

```diff
@@ -430,9 +434,13 @@ CREATE TABLE information_schema.table_constraints (
 					defString,                         // constraint_catalog
 					parser.NewDString(db.Name),        // constraint_schema
 					dStringOrNull(name),               // constraint_name
+					defString,                         // table_catalog
 					parser.NewDString(db.Name),        // table_schema
 					parser.NewDString(table.Name),     // table_name
 					parser.NewDString(string(c.Kind)), // constraint_type
+					yesOrNoDatum(c.IsDeferrable),      // is_deferrable
+					yesOrNoDatum(c.InitiallyDeferred), // initially_deferred
```

Comment:
You can just hardcode the argument here as false, there's a handful of places we do something similar when we need to expose info regarding features we don't support (also nit: extra line).

Reply:
Fixed.